homeassistant/components/myuplink/quality_scale.yaml

```diff
@@ -0,0 +1,99 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      No custom actions are defined.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
For reauth, we should also do a test where we add a mock config entry with a unique_id and then try to reauthenticate with a different account that would raise the mismatch error

Reply:
Done